.pre-commit-config.yaml

```diff
@@ -1 +1,46 @@
-repos: []
+# To enable this pre-commit hook run:
+# `pip install pre-commit` or `brew install pre-commit`
+# Then run `pre-commit install`
+
+# Learn more about this config here: https://pre-commit.com
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: check-toml
+      - id: check-yaml
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: requirements-txt-fixer
+      #- id: name-tests-test
+      #  args:
+      #    - --django
+      #- id: trailing-whitespace
+      #  args:
+      #    - --markdown-linebreak-ext=md
```

Comment:
We want these lines but adding them creates too much confusion so that pull requests never get merged so let's remove them so that we forget to fix them later.